cmd/cli/devstack/devstack.go

```diff
@@ -206,12 +207,39 @@ func runDevstack(cmd *cobra.Command, cfg types.BacalhauConfig, fsr *repo.FsRepo,
 		devstack.WithComputeConfig(computeConfig),
 		devstack.WithRequesterConfig(requesterConfig),
 	)
+	userKeyPath, err := fsr.UserKeyPath()
+	if err != nil {
+		return err
+	}
+	userKey, err := baccrypto.LoadUserKey(userKeyPath)
+	if err != nil {
+		return err
+	}
```

Comment:
do we really need to pass userKey? Isn't it deprecated?

Reply:
Yes, its required by the authentication service, and removing it is out of scope here.